chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -826,6 +826,9 @@ searchConditions.programmingLanguage	= Programming Language
 searchConditions.fileTypeID		= Attachment File Type
 searchConditions.annotationText = Annotation Text
 searchConditions.annotationComment = Annotation Comment
+searchConditions.author = Author
+searchConditions.editor = Editor
+searchConditions.bookAuthor = Book Author
```

Comment:
We already have these, so let's just add a `switch` to `SearchConditions.getLocalizedName()` to check for these and use `Zotero.CreatorTypes.getLocalizedString()` instead.